compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -35,12 +70,35 @@ void buildAMDAIETransformPassPipeline(OpPassManager &pm) {
   // efforts. Once the C++ passes are ready, we will include these passes by
   // default and take away the guarding.
   if (clUseCPlusPlusTransformPasses) {
+    pm.addPass(createCleanupPass());
+    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createCSEPass());
     pm.addPass(createAMDAIETileAndFusePass(1));
-    pm.addPass(createAMDAIEPadAndBufferizePass());
+    pm.addPass(createAMDAIEPadAndBufferizePass(1));
+    pm.addPass(createCleanupPass());
+    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createCSEPass());
+    pm.addPass(createAMDAIETileAndFusePass(2));
+    pm.addPass(createAMDAIEPadAndBufferizePass(2));
+    pm.addPass(createCleanupPass());
+    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createCSEPass());
+    pm.addPass(createAMDAIETileAndFusePass(3));
+    pm.addPass(createCleanupPass());
+    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createCSEPass());
+    pm.addPass(createAMDAIEPadAndBufferizePass(3));
+    pm.addPass(createCleanupPass());
+    pm.addPass(createCanonicalizerPass());
+    pm.addPass(createCSEPass());
+    pm.addPass(createAMDAIEEliminateEmptyTensorPass());
```

Comment:
Isnt the `EliminateEmptyTensorsPass` already run as part of addIREEComprehensiveBufferizePasses. Do we need to run this pass at all 

https://github.com/openxla/iree/blob/914f30641f3c41338f8a7820711743ca49a2a76f/compiler/src/iree/compiler/Codegen/Common/IREEComprehensiveBufferizePass.cpp#L249